src/lib/media/video/compress.web.ts

```diff
@@ -5,13 +5,13 @@ const MAX_VIDEO_SIZE = 1024 * 1024 * 100 // 100MB
 
 // doesn't actually compress, but throws if >100MB
 export async function compressVideo(
-  file: string,
+  asset: {uri: string},
```

Comment:
This'll work, but ImagePickerAsset  would be more consistent

Reply:
Does it actually have all the types from ExpoVideo?

Reply:
changed it, but no idea if all those values are actually available. guess thats why most are optional